features/registration_feature.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Registration page
+  As a customer
+  In order to login
+  I need to be able to register
+
+  Scenario: Visitor sees a form to register on register page
+    #Given there is a register page
+    When I visit the register page
+    Then I would like to see a form for registration
+    And I would like to see a text field for user name
+    And I would like to see a text field for password
+    And I would like to see a text field for password confirmation
+    And I would like to see a register button
+
+  Scenario: Visitor writes his information and presses register
+    Given username and password has been entered
```

Comment:
I would like you to split this step up into separate steps with dynamic values
```
Given I fill in "Name" with "Alfred"
```
Also, you need to navigate to the signup page as the first step.

Reply:
Done